utils/zerotrie/src/options.rs

```diff
@@ -118,6 +118,7 @@ impl<S: ?Sized> ZeroTrieWithOptions for crate::ZeroTrieSimpleAscii<S> {
 
 impl<S: ?Sized> crate::ZeroTrieSimpleAscii<S> {
     #[cfg(feature = "serde")]
+    #[allow(const_evaluatable_unchecked)]
```

Comment:
I was getting a really strange error here

```
error: cannot use constants which depend on generic parameters in types
   --> utils/zerotrie/src/options.rs:121:5
    |
121 |     pub(crate) const FLAGS: u8 = Self::OPTIONS.to_u8_flags();
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #76200 <https://github.com/rust-lang/rust/issues/76200>
    = note: `-D const-evaluatable-unchecked` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(const_evaluatable_unchecked)]`
```